src/server/rpc/procedures/geolocation/geolocation.js

```diff
@@ -135,7 +140,14 @@ if(!process.env.GOOGLE_GEOCODING_API) {
             });
     };
 
-    // find places near a coordinate (20 reults max)
+    /**
+     * find places near a coordinate (20 reults max)
```

Comment:
typo: `reults`

I would also consider being more clear about things like "coordinate" because that doesn't have a lot of meaning in the current form (especially when the target audience is young).